### DISCUSSION TITLE
[Show and Tell] <zkBitcoin Token> Proof of work minting on the zkSync Era blockchain based on Bitcoin

### DISCUSSION THREAD

**Discussion by ZKBitcoinToken** [zkSync-Community-Hub/zksync-developers#455] [Show and Tell]
### Project Name

zkBitcoin

### Repo Link

https://github.com/zkBitcoinToken

### Demo Link (Optional)

https://zkBitcoin.org

### Technologies Used

Decentralized mining using zkSync Era

### Describe your project. What does it do? Why did you create it? Any interesting challenges you faced?

zkSync Era with EIP4844 became a much cheaper blockchain, so to take advantage of the low fee blockchain, a proof of work token was always wanted to be on a Layer 2 but costs made it too expensive and launches were often full of failed transactions.  In steps zkBitcoin, it allows up to 2048 single answers at once, so launches are full of accepted answers instead of failed transactions.
It uses a zkSync Era layer 2 recent blockhash as the Challenge computers must solve to get a reward. 

### What specific feedback are you looking for?

Gas improvements on the contract.  Any technical website or other feedback.

### How can the community help or contribute to your project?

By reaching a large audience, zkBitcoin will become more decentralized and a better store of value over time with the more users that hold zkBitcoin expecting value to raise using Bitcoins store of value model.

### Additional Information (Optional)

Offical Website: https://zkbitcoin.org/
Whitepaper: https://zkbitcoin.org/whitepaper.html
Discord: https://discord.gg/dNKbmFvvJU
Twitter: https://twitter.com/zkBitcoinToken

Explorers:
https://era.zksync.network/address/0x366d17aDB24A7654DbE82e79F85F9Cb03c03cD0D
https://explorer.zksync.io/address/0x366d17aDB24A7654DbE82e79F85F9Cb03c03cD0D

Miners:
https://github.com/ZKBitcoinToken/Windows-CPU-GPU-zkBitcoin-Miner
https://github.com/ZKBitcoinToken/Linux-CPU-GPU-zkBitcoin-Miner

**Comment by bxpana** [comment]
Appreciate you sharing with the community.
